packages/dx-react-chart/docs/reference/zoom-and-pan.md

```diff
@@ -23,17 +23,17 @@ import { ZoomAndPan } from '@devexpress/dx-react-chart';
 
 Name | Type | Default | Description
 -----|------|---------|------------
-defaultViewport? | [ZoomAndPan.Viewport](#zoomandpanviewport) | The default viewport.
-viewport? | [ZoomAndPan.Viewport](#zoomandpanviewport) | The viewport.
-onViewportChange? | (viewport: [ZoomAndPan.Viewport](#zoomandpanviewport)) => void | A function that is executed when the viewport changes.
+defaultViewport? | [Viewport](#viewport) | The default viewport.
+viewport? | [Viewport](#viewport) | The viewport.
+onViewportChange? | (viewport: [Viewport](#viewport)) => void | A function that is executed when the viewport changes.
 interactionWithArguments? | 'none' &#124; 'pan' &#124; 'zoom' &#124; 'both' | The type of interaction available for the argument scale.
 interactionWithValues? | 'none' &#124; 'pan' &#124; 'zoom' &#124; 'both' | The type of interaction available for the value scale.
 zoomRegionKey? | 'shift' &#124; 'alt' &#124; 'ctrl' | The key the allows a user to zoom a region by selecting it with the mouse drag gesture.
 dragBoxComponent? | ComponentType&lt;[ZoomAndPan.DragBoxProps](#zoomandpandragboxprops)&gt; | A component that renders the rectangle that appears when a user zooms a region.
 
 ## Interfaces
 
-### ZoomAndPan.Viewport
+### Viewport
```

Comment:
In most files we use the rule `Plugin_Name.Interface_Name` for declaring interfaces.

Reply:
`PluginName` prefix affects generated ts code. The following is generated

```ts
export namespace ZoomAndPan {
  /** The viewport configuration. */
  export type Viewport = ZoomAndPanBase.Viewport;
}
```

and causes an error as there is no `Viewport` field in `ZoomAndPanBase`.